__tests__/app.test.js

```diff
@@ -421,22 +421,47 @@ describe("GET: /api/articles (topic query)", () => {
       .get("/api/articles?search=topic")
       .expect(200)
       .then((result) => {
-        const articles = result.body
-          expect(articles).toHaveLength(13);
-        });
+        const articles = result.body;
+        expect(articles).toHaveLength(13);
       });
+  });
   test("404: if a topic is searched for that doesn't exist", () => {
     return request(app)
       .get("/api/articles?search=topic&&topic=elephant")
       .expect(404)
-      .then((result)=>{
-          expect(result.body.msg).toEqual("there are no topics called elephant");
-        });
-      })
-    })
-describe('GET: /api/articles/:article_id (comment_count) adding a query to the endpoint',() =>{
-  test("200: returns an object which has the sum of the comment_count for that article_id", ()=>{
+      .then((result) => {
+        expect(result.body.msg).toEqual("there are no topics called elephant");
+      });
+  });
+});
+describe("GET: /api/articles/:article_id (comment_count) adding a query to the endpoint", () => {
+  test("200: returns an object which has the sum of the comment_count for that article_id", () => {
     return request(app)
-    .get('/api/articles/1?sum_of=comment_count')
+      .get("/api/articles/1?sum_of=comment_count")
```

Comment:
This isn't quite right. The task is just looking for the comment_count property to have been added to all the article objects. It is **not** a query.

To pass the test an update to the query is all that is required - to add that new property. It is achievable with just a single query.

Refactor the test and then refactor your solution 👍